generators/generator-microsoft-bot-adaptive/generators/app/index.js

```diff
@@ -223,6 +230,22 @@ module.exports = class extends BaseGenerator {
     );
   }
 
+  _writeBotProject() {
+    const { botName, botProjectSettings } = this.options;
+
+    const botProject = this.fs.readJSON(
+      this.templatePath('assets', 'botName.botproj')
+    );
+
+    botProject.name = botName;
+    botProject.skills = (botProjectSettings && botProjectSettings.skills) || {};
```

Comment:
Due to the runtypes definition, `botProjectSettings` will always be at least `{}`, so no need to be too careful here.